CHANGES.md

```diff
@@ -81,6 +82,8 @@ To be released.
     an empty `TxIds` from peers, and it had made the network waste bandwidth for
     unnecessary messages.  `Swam` became to no more send such empty `GetTxs`.
     [[#297]]
+ -  `BlockChain<T>.Swap()` became to delete an index, tx nonces and state
```

Comment:
It's trivial, [but…][1] 🙄

```suggestion
 -  `BlockChain<T>.Swap()` became to delete an index, tx nonces, and state
```

[1]: https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/